source/features/dim-bots.tsx

```diff
@@ -50,17 +55,21 @@ function undimBots(event: DelegateEvent): void {
 	resetScroll();
 }
 
-function init(signal: AbortSignal): void {
-	for (const bot of $$(commitSelectors)) {
-		// Exclude co-authored commits
-		if ($$('a', bot.parentElement!).every(link => link.matches(commitSelectors))) {
-			bot.closest('.commit, .Box-row')!.classList.add(dimBots.class);
-		}
-	}
+function dimCommit(commit: HTMLElement): void {
+	commit.closest([
+		'.listviewitem',
+		'.Box-row', // Old view style before Nov 2023
```

Comment:
I dropped .commit because it was being carried over from the initial PR many years ago